Selenium.Axe/Selenium.Axe.Test/AxeBuilderTest.cs

```diff
@@ -1,20 +1,20 @@
 ﻿using FluentAssertions;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
 using Moq;
 using Newtonsoft.Json;
+using NUnit.Framework;
 using OpenQA.Selenium;
 using System;
 using System.Collections.Generic;
 using System.Collections.ObjectModel;
 
 namespace Selenium.Axe.Test
 {
-    [TestClass]
+    [TestFixture]
     public class AxeBuilderTest
     {
-        private Mock<IWebDriver> webDriverMock;
-        private Mock<IJavaScriptExecutor> jsExecutorMock;
-        private Mock<ITargetLocator> targetLocatorMock;
+        private static Mock<IWebDriver> webDriverMock;
```

Comment:
Did you find that these mocks were particularly expensive to set up for some reason? If it isn't a significant perf difference, I'd recommend keeping the mocks and their initialization per-test rather than one-time, for better test isolation